tests/testthat/test-filters.R

```diff
@@ -98,6 +98,15 @@ with_mock_crunch({
             '{"variable":"https://app.crunch.io/api/datasets/1/variables/gender/"},',
             '{"value":1}]}}')
     })
+    
+    test_that("Alter  a filter by [[<-", {
+        expect_PATCH(filters(ds)[["Occasional Political Interest"]] <- ds$gender=="Female",
```

Comment:
Human linter asks: can we put spaces around operators like this `==` and the `~` and `+` in the formulas?